platform-api/che-core-api-account/src/main/java/org/eclipse/che/api/account/shared/dto/AccountDescriptor.java

```diff
@@ -44,6 +46,12 @@
 
     AccountDescriptor withId(String id);
 
+    List<UsersWorkspaceDto> getWorkspaces();
```

Comment:
Account is a class and i don't want to separate it on interface and impl, at least not now